<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddPioneer</class>
 <widget class="QDialog" name="AddPioneer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Pioneer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/Images/add_pio.png</normaloff>:/icons/Images/add_pio.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_add_pio">
     <property name="text">
      <string>&lt;h3&gt;Add Pioneer &lt;/h2&gt;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_name">
     <property name="text">
      <string>Name: </string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLabel" name="label_name_error">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="3" colspan="2">
    <widget class="QLabel" name="label_birth_year">
     <property name="text">
      <string>Birth Year:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QLabel" name="label_byear_error">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QLineEdit" name="input_name"/>
   </item>
   <item row="2" column="3" colspan="3">
    <widget class="QLineEdit" name="input_bYear"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_sex">
     <property name="text">
      <string>Sex:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QLabel" name="label_sex_error">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="3" colspan="2">
    <widget class="QLabel" name="label__death_year">
     <property name="text">
      <string>Death Year:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QLineEdit" name="input_sex"/>
   </item>
   <item row="4" column="3" colspan="3">
    <widget class="QLineEdit" name="input_dYear"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_description">
     <property name="text">
      <string>Description:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="5">
    <widget class="QLabel" name="label_description_error">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="6">
    <widget class="QLineEdit" name="input_description"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_relations">
     <property name="text">
      <string>Relations:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" rowspan="2" colspan="2">
    <widget class="QListWidget" name="list_unrelated_computers"/>
   </item>
   <item row="8" column="2" colspan="2">
    <widget class="QPushButton" name="button_add_relations">
     <property name="text">
      <string>Add Relation</string>
     </property>
    </widget>
   </item>
   <item row="8" column="4" rowspan="2" colspan="2">
    <widget class="QListWidget" name="list_related_computers"/>
   </item>
   <item row="9" column="2" colspan="2">
    <widget class="QPushButton" name="button_remove_relation">
     <property name="text">
      <string>Remove Relation</string>
     </property>
    </widget>
   </item>
   <item row="10" column="2" colspan="2">
    <widget class="QPushButton" name="button_add_pioneer">
     <property name="text">
      <string>Add Pioneer</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>input_name</tabstop>
  <tabstop>input_sex</tabstop>
  <tabstop>input_bYear</tabstop>
  <tabstop>input_dYear</tabstop>
  <tabstop>input_description</tabstop>
  <tabstop>button_add_pioneer</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
